tests/operator_test.go

```diff
@@ -960,70 +1007,62 @@ spec:
 			}
 		})
 
-		It("[test_id:835]Ensure infra can start with DataVolume feature gate enabled.", func() {
+		It("Ensure infra can handle dynamically detecting DataVolume Support", func() {
 			if !tests.HasDataVolumeCRD() {
 				Skip("Can't test DataVolume support when DataVolume CRD isn't present")
 			}
+			tests.SkipIfVersionBelow("Skipping dynamic cdi test in versions below 1.13 because crd garbage collection is broken", "1.13")
 
 			// This tests starting infrastructure with and without the DataVolumes feature gate
-			running := false
 			vm = tests.NewRandomVMWithDataVolume(tests.AlpineHttpUrl, tests.NamespaceTestDefault)
+			running := false
 			vm.Spec.Running = &running
 
-			// Disable the gate
-			By("DisablingFeatureGate")
-			tests.DisableFeatureGate("DataVolumes")
-
-			// Cycle the infrastructure to pick up the change.
-			allPodsAreReady(originalKv)
-			sanityCheckDeploymentsExist()
-
-			By("Deleting KubeVirt object")
-			deleteAllKvAndWait(false)
+			// Delete CDI object
```

Comment:
genuine newbie question: does ginkgo execute tests / suites in parallel? if so, deleting CDI may affect other tests running in parallel.

Reply:
ginkgo can run tests in parallel, We run in series though because of issues like this.